dockers/docker-sonic-telemetry/telemetry.sh

```diff
@@ -46,6 +46,7 @@ else
 	TELEMETRY_ARGS+=" -v=2"
 fi
 
-exec /usr/sbin/telemetry ${TELEMETRY_ARGS}
+cd /usr/sbin
+exec telemetry ${TELEMETRY_ARGS}
```

Comment:
why "cd" is required?

Reply:
translib (which telemetry uses) looks for the schema directory in the CWD.

Reply:
Sounds like a little fragile approach. It would be nice to find some solid approach.

Being inside docker, the local path can be hardcoded or defined in some .rc or ...

Reply:
Sure, but this would be a change in translib.

Reply:
is this going to be fixed in this PR?

Reply:
no it would need to be fixed in translib which is in sonic-mgmt-framework repo.

Reply:
Actually, looking at the sonic-mgmt-framework startup scripts, there is a env variable defined: `CVL_SCHEMA_PATH=/usr/sbin/schema` so I will use that.

Reply:
https://github.com/project-arlo/sonic-buildimage/commit/54b6eb1b74a4d005da42417ab8755a311a362f06

Reply:
Forgot to export it.. https://github.com/project-arlo/sonic-buildimage/commit/a4e924ae523eb545fbfdb0f3e2f3ee601d6615ef